src/ExperimentTable.js

```diff
@@ -0,0 +1,174 @@
+
+    const dataArray = selectedSearch ?
+      this.sort(aaData).filter(data => data && Object.values(data).some(value => value.toString().toLowerCase().includes(selectedSearch.toLowerCase()))) :
+      this.filter(this.sort(aaData), tableHeader).filter(data => selectedKingdom ? data.kingdom === selectedKingdom : true)
+    const currentPageData = entryPerPage ? dataArray.slice(entryPerPage * (currentPage - 1), entryPerPage * currentPage) : dataArray
+    const kingdomOptions = [...new Set(aaData.map(data => data.kingdom ))]
+    const entriesPerPageOptions = [10, 25, 50]
+
+    return (
+      <div className={`row expanded`}>
+        <div className={`row expanded`}>
+          <div className={`small-12 medium-4 large-2 columns`}>
+            <label> Kingdom:
+              <select defaultValue={``} className={`kingdom`} onChange={event => this.setState({selectedKingdom: event.target.value})}>
+                <option value={``} >All</option>
+                {kingdomOptions.map(kingdom => <option value={kingdom}>{kingdom.charAt(0).toUpperCase() + kingdom.slice(1)}</option>)}
+              </select>
+            </label>
+          </div>
+          <div className={`small-12 medium-4 large-2 columns`}>
+            <label>Entries per page:
+              <select defaultValue={10} onChange={this.setValue(`selectedNumber`)}>
+                { entriesPerPageOptions.map(entries => aaData.length >= entries ? <option value={entries}>{entries}</option> : []) }
+                <option value={aaData.length}>All</option>
+              </select>
+            </label>
+          </div>
+          <div className={`large-2 columns`}>
+            <label>Search all columns:
+              <input type={`search`} placeholder={`Type here ...`}
+                onChange={this.setValue(`selectedSearch`)}/></label>
+          </div>
+        </div>
+        <div className={`small-12 columns`} >
+          <Table border>
+            <Table.Head>
+              {enableIndex && <Table.TextHeaderCell key={`index`} flexBasis={100} flexShrink={100} flexGrow={100}>Index</Table.TextHeaderCell>}
+              {tableHeaderCells(tableHeader, searchedColumn, searchQuery, orderedColumn, ordering,
+                (columnNumber) =>
+                  this.setState({
+                    orderedColumn: columnNumber,
+                    ordering: !this.state.ordering}),
+                (value, columnNumber) =>
+                  this.setState({
+                    searchQuery: value,
+                    searchedColumn: columnNumber
+                  })
+              )}
+              {
+                enableDownload && <Table.TextHeaderCell className={`downloadHeader`} flexBasis={100} flexShrink={100} flexGrow={100}>
+                  {
+                    checkedArray.length > 0 ?
+                      <a href={`${host}experiments/download/zip?${checkedArray.map(accession => `accession=${accession}`).toString().replace(`,`,`&`)}`}>
```

Comment:
You have this habit of manipulating strings to build links. You should always use a library to build URLs. We include [`urijs`](https://medialize.github.io/URI.js/) in all packages for this purpose. It might be cleaner to have a function that builds the URL from `checkedArray`.